spyrograph/hypotrochoid/hypocycloid.py

```diff
@@ -9,9 +9,21 @@
 from spyrograph._cycloid import _Cycloid
 
 class Hypocycloid(_Cycloid, Hypotrochoid):
-    """Model of a hypocycloid which is a special case of a hypotrochoid where
-    the circle is rolling around the inside of the fixed circle and has 1/3
-    the radius of the fixed circle
+    """
+    Model of a hypocycloid, which is a mathematical curve traced by a point on a circle as
+    it rolls around the inside of another fixed circle. This class represents a special case of
+    a hypotrochoid, where the distance from the point to the rolling circle is equal to the radius
+    of the rolling circle.
+
+    The Hypocycloid class allows you to generate points along the curve using mathematical
+    equations and visualize the curve in a plot. It also provides properties and methods for
+    computing the length, area, and curvature of the curve, as well as for plotting and animating
+    the curve.
```

Comment:
Please remove the "for computing the length, area, and curvature of the curve, as well as" as Hypocycloid currently doesn't have this behavior